src/wallet/rpcdump.cpp

```diff
@@ -819,9 +819,97 @@ UniValue dumpwallet(const JSONRPCRequest& request)
     return reply;
 }
 
+struct ImportData
+{
+    // Input data
+    std::unique_ptr<CScript> redeemscript; //! Provided redeemScript; will be moved to `import_scripts` if relevant.
+    std::unique_ptr<CScript> witnessscript; //! Provided witnessScript; will be moved to `import_scripts` if relevant.
```

Comment:
For members, doxygen uses `//!<`
http://www.doxygen.nl/manual/docblocks.html#memberdoc